extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/sender/VertxHttpSender.java

```diff
@@ -101,19 +101,33 @@ public void send(Marshaler marshaler,
             return;
         }
 
+        String marshalerType = marshaler.getClass().getSimpleName();
         String requestURI = basePath + signalPath;
         var clientRequestSuccessHandler = new ClientRequestSuccessHandler(client, requestURI, headers, compressionEnabled,
                 contentType,
                 contentLength, onHttpResponseRead,
                 onError, marshaler, 1, isShutdown::get);
-        initiateSend(client, requestURI, MAX_ATTEMPTS, clientRequestSuccessHandler, onError, isShutdown::get);
+        initiateSend(client, requestURI, MAX_ATTEMPTS, clientRequestSuccessHandler, new Consumer<>() {
+            @Override
+            public void accept(Throwable throwable) {
+                failOnClientRequest(marshalerType, throwable, onError);
+            }
+        });
+    }
+
+    private void failOnClientRequest(String type, Throwable t, Consumer<Throwable> onError) {
+        String message = "Failed to export "
+                + type
+                + "s. The request could not be executed. Full error message: "
```

Comment:
Very minor but given the point of this PR is to add a message: I would drop the `s` from here. Won't work for `Child` for instance.